charts/harvester-node-manager/templates/daemonset.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ include "harvester-node-manager.name" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "harvester-node-manager.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "harvester-node-manager.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+    {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      labels:
+        {{- include "harvester-node-manager.selectorLabels" . | nindent 8 }}
+        name: harvester-node-manager
+    spec:
+      serviceAccountName: {{ include "harvester-node-manager.name" . }}
+      containers:
+        - name: node-manager
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: IfNotPresent
+          command:
+            - harvester-node-manager
+          env:
+            - name: NODENAME
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: spec.nodeName
+            - name: HOST_PROC
+              value: /host/proc
+          securityContext:
+            privileged: true
```

Comment:
Is there a way to not run a privileged container? Perhaps just adding required caps.
Doing a search and seems `procMount`  could be helpful, but it doesn't cover sysfs.

Reply:
We can use privileged until we find a better solution. Thanks!